packages/core/app/initializers/spree-ember-core.js

```diff
@@ -14,11 +13,6 @@ export function initialize(container, application) {
   application.inject('route', 'spree', 'service:spree');
   application.inject('controller', 'spree', 'service:spree');
   application.inject('component', 'spree', 'service:spree');
-
-  /* Copy Environment and Spree Configuration to Spree Service */
-  var SpreeService = container.lookup('service:spree');
-  SpreeService.set('environment', ENV.environment);
-  SpreeService.set('config', ENV['spree'] || {});
 }
```

Comment:
I'm almost done with instance-initializers on my local - so I'm happy to finish this off.  Would love to have this PR in first though!